src/Bicep.Core.Samples/Files/baselines_bicepparam/External_Inputs/parameters.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "singleParam": {
+      "expression": "[externalInputs('0')]"
+    },
+    "singleParamCast": {
+      "expression": "[bool(externalInputs('1'))]"
+    },
+    "foo": {
+      "expression": "[externalInputs('2')]"
+    },
+    "bar": {
+      "expression": "[externalInputs('3')]"
+    },
+    "baz": {
+      "expression": "[externalInputs('4')]"
+    },
+    "varRef": {
+      "expression": "[bool(externalInputs('5'))]"
+    },
+    "objectConfig": {
+      "expression": "[externalInputs('6')]"
+    },
+    "c": {
+      "expression": "[externalInputs('7')]"
+    },
+    "a1": {
+      "expression": "[externalInputs('8')]"
+    },
+    "b1": {
+      "expression": "[externalInputs('8')]"
+    },
+    "c1": {
+      "expression": "[externalInputs('8')]"
+    },
+    "first": {
+      "expression": "[int(externalInputs('9'))]"
+    },
+    "second": {
+      "expression": "[externalInputs('10')]"
+    },
+    "result": {
+      "expression": "[format('{0} combined with {1}', int(externalInputs('9')), externalInputs('10'))]"
+    }
+  },
+  "externalInputDefinitions": {
+    "2": {
```

Comment:
What are your thoughts on using the "kind" name for name generation, to make the understanding of the generated code + errors a bit more user-friendly - e.g.:

`sys_cli_2` instead of just `2`

Reply:
Yeah totally, makes sense. Will update.

Reply:
updated